app/controllers/users_controller.rb

```diff
@@ -136,11 +136,11 @@ def create_params
   end
 
   def edit_user_params
-    @user.admin_editable? ? params.require(:user).permit(:email, :first_name, :last_name, :username) : empty_params
+    @user.admin_editable? ? params.require(:user).except(:internal).permit(:email, :first_name, :last_name, :username) : with_dropped_params { empty_params }
```

Comment:
This seems like an odd construct. Why do we need to wrap `with_dropped_params { empty_params }` when the params should have nothing inside them?